### DISCUSSION TITLE
Welcome New Members! 👋

### DISCUSSION THREAD

**Discussion by NS007-dev** [App-Choreography/OurContributors#32] [Team Posts]
Welcome to our new App-Choreographers: 
- @dreamboat999 
- @HarmonySkull 
- @SoftwareArchitect9817 
- @SimonBaey

**Comment by HarmonySkull** [comment]
Thank you very much, I hope to be able to collaborate actively.

**Comment by NS007-dev** [comment]
Feel free to browse through our repos!
